tsdb/chunks/head_chunks_test.go

```diff
@@ -188,51 +185,41 @@ func TestHeadReadWriter_Truncate(t *testing.T) {
 		return mint
 	}
 
-	cutFile := func(i int) {
+	cutFile := func(i int) int64 {
```

Comment:
`cutFile` is now so short that I think it's less mental overhead if you inline it.

Reply:
In any case, the `i` parameter is not used anymore.